tests/lib/Integration/Traits/PagerfantaFindTraitTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Netgen\Bundle\EzPlatformSiteApiBundle\Tests\lib\Integration\Traits;
+
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use Netgen\EzPlatformSiteApi\Core\Site\Pagination\Pagerfanta\ContentSearchAdapter;
+use Netgen\EzPlatformSiteApi\Core\Site\Pagination\Pagerfanta\ContentSearchHitAdapter;
+use Netgen\EzPlatformSiteApi\Core\Site\Pagination\Pagerfanta\LocationSearchHitAdapter;
+use Netgen\EzPlatformSiteApi\Core\Site\Pagination\Pagerfanta\LocationSearchAdapter;
+use Netgen\EzPlatformSiteApi\Core\Traits\PagerfantaFindTrait;
+use Netgen\EzPlatformSiteApi\Tests\Integration\BaseTest;
+
+class PagerfantaFindTraitTest extends BaseTest
+{
+    use PagerfantaFindTrait;
```

Comment:
This feels wrong. A test class that uses the trait under test instead of instantiating a (stub) object that uses the trait?